src/plugins/intel_npu/src/al/include/intel_npu/config/npuw.hpp

```diff
@@ -17,6 +17,7 @@ namespace intel_npu {
 //
 
 void registerNPUWOptions(OptionsDesc& desc);
+void registerNpuwLlmOptions(OptionsDesc& desc);
```

Comment:
These two lines' formatting choices scream at each other

Reply:
It was done just because `registerNPUWLLMOptions` looks a bit hard to parse, so I decided to go with `registerNpuwLlmOptions` and then somewhere in future refactor the first part

Reply:
Fixed